File: mario-more/mario.c
#include "cs50.h"
#include "stdio.h"

int main(void)
{
    const int MIN_HEIGHT = 1;
    const int MAX_HEIGHT = 8;

    int height = get_int("How tall should the pyramid be? ");

    while (height < MIN_HEIGHT || height > MAX_HEIGHT)
    {
        height = get_int("How tall should the pyramid be? ");
    }

    for (int i = 0; i < height; i++)
    {
        for (int j = 0; j < height - i - 1; j++)
        {
            printf(" ");
        }

        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }

        printf("  ");

        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int


def main():
    while True:
        height = get_int("Height: ")
        if height >= 1 and height <= 8:
            break

    for i in range(height):
        for _ in range(height - i - 1):
            print(" ", end='')
        for _ in range(i + 1):
            print("#", end='')

        print("  ", end='')

        for _ in range(i + 1):
            print("#", end='')

        print("")


main()
